### DISCUSSION TITLE
Unclear messaging around fonts

### DISCUSSION THREAD

**Discussion by maroider** [starship/starship#1425] [General]
I'm having trouble understanding which set of fonts it is I'm supposed to install.
There are two places on the website where I'm told to install a set of fonts. Once in the __Installation__ section and once under __Frequently Asked Questions__.

> # Installation
> ## Prerequisites
> 
> * A Nerd Font installed and enabled in your terminal (for example, try the Fira Code Nerd Font ).

> # Frequently Asked Questions
> ## Why don't I see a glyph symbol in my prompt?
> The most common cause of this is system misconfiguration. Some Linux distros in particular do not come with font support out-of-the-box. You need to ensure that:
> [...]
> * You are using a powerline-patched font .

So which is it? Do I just pick one? Should I install both?

**Comment by davidkna** [comment]
Nerd fonts carry various patches,  one of which is the powerline patch, so any nerd font is also a powerline font.
If you want to use the [nerd font present](https://starship.rs/presets/#nerd-font-symbols) you will need a nerd font.
Unless something changed, a powerline font should be sufficient if you're not using the preset.
